FitGoal/FitGoal/ExerciseDetailViewController/ExerciseDetailViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ExerciseDetailViewController.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 13/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class ExerciseDetailViewController: UIViewController {
+    
+    var exercise: Exercise?
+    
+    private var startExerciseButton: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = #colorLiteral(red: 0.56, green: 0.07, blue: 1, alpha: 1)
+        let string = "Start Exercise".formattedText(
+            font: "Roboto-Regular",
+            size: 17,
+            color: .white,
+            kern: 0
+        )
+        button.setAttributedTitle(string, for: .normal)
+        
+        return button
+    }()
+    
+    private let exerciseCollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.addSubview(exerciseCollectionView)
+        view.addSubview(startExerciseButton)
+        
+        self.view.backgroundColor = #colorLiteral(red: 0.9647058824, green: 0.9647058824, blue: 0.9647058824, alpha: 1)
+        
+        self.exerciseCollectionView.dataSource = self
+        self.exerciseCollectionView.delegate = self
+        self.exerciseCollectionView.alwaysBounceVertical = true
+        self.exerciseCollectionView.backgroundColor = .clear
+        
+        exerciseCollectionView.register(ExercisePreviewCell.self, forCellWithReuseIdentifier: ExercisePreviewCell.identifier)
+        
+        exerciseCollectionView.register(ExerciseDescriptionCell.self, forCellWithReuseIdentifier: ExerciseDescriptionCell.identifier)
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        exerciseCollectionView.frame = view.bounds
+        startExerciseButton.frame = CGRect (x: 16, y: view.bounds.maxY - 84, width: view.bounds.width - 32, height: 54)
+        startExerciseButton.layer.cornerRadius = startExerciseButton.bounds.height/2
+    }
+}
+
+extension ExerciseDetailViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        guard let exerciseSection = ExerciseDetailSection(rawValue: section) else {
+            fatalError("Section value should have a corresponding case in the HomeSection enum")
```

Comment:
The error message is outdated since it doesn't refer to the `HomeSection` enum in this case.